docs/static/advanced-pipeline.asciidoc

```diff
@@ -117,15 +118,8 @@ filter {
     }
 }
 --------------------------------------------------------------------------------
-And change output to see a result :
-[source,json]
---------------------------------------------------------------------------------
-output {
-  stdout { codec => rubydebug }
-}
---------------------------------------------------------------------------------
```

Comment:
Did you mean to delete this section? Above, your addition targets the filter section but without this below the user will see nothing happening.
```
Edit the `first-pipeline.conf` file and replace the entire `output` section with the following text:

[source,json]
---------------------------------------------------------------------------------
output {
  stdout { codec => rubydebug }
}
 --------------------------------------------------------------------------------

The `codec => rubydebug` will output a formatted representation of each event to the terminal.
This allows you to view the results of any changes to you make to the config as you work though
this tutorial.
You may wish to switch back to this output config from time to time to visualize your changes.
```